ogn/parser/pattern.py

```diff
@@ -37,12 +38,15 @@
     RAM:(?P<ram_free>[\d.]+)\/(?P<ram_total>[\d.]+)MB\s
     NTP:(?P<ntp_offset>[\d.]+)ms\/(?P<ntp_correction>[+-][\d.]+)ppm\s?
     (?:(?P<cpu_temperature>[+-][\d.]+)C\s*)?
+    (?:(?P<aircraft_counter_visible>\d+)\/(?P<aircraft_counter_total>\d+)Acfts\[1h\]\s*)?
```

Comment:
I see, I'll look into it tomorrow. Do you have a reliable source to see the protocol changes from one release to the other?